dockerfiles/Dockerfile

```
@@ -11,7 +11,7 @@
 # docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" -f ./dockerfiles/Dockerfile.multiarch --build-arg FLB_TARBALL=https://github.com/fluent/fluent-bit/archive/v1.8.11.tar.gz ./dockerfiles/
 
 # Set this to the current release version: it gets done so as part of the release.
-ARG RELEASE_VERSION=2.1.9
+ARG RELEASE_VERSION=2.1.100
```

Not sure why this picked 100...
```suggestion
ARG RELEASE_VERSION=2.1.10
```